packages/core/lib/generators/integrations/svg_integration.dart

```diff
@@ -64,9 +68,7 @@ ${isPackage ? "\n  static const String package = '$packageName';" : ''}
       semanticsLabel: semanticsLabel,
       excludeFromSemantics: excludeFromSemantics,
       theme: theme,
-      colorFilter: colorFilter,
-      color: color,
-      colorBlendMode: colorBlendMode,
+      colorFilter: colorFilter ?? (color == null ? null : ColorFilter.mode(color, colorBlendMode)),
```

Comment:
Do not imply the `colorFilter`, leave all arguments delivered as-is.

Reply:
default SvgPicture constructor does not support color and colorBlendMode

Reply:
Then we can ignore that usage or try to raise warnings.

Reply:
both `color` and `colorBlendMode` already marked as deprecated